docs/settings.md

```diff
@@ -1,10 +0,0 @@
-## Settings
-
-`JQUERY_URL`
```

Comment:
Rather than delete this file, we should mention `SMART_SELECTS_URL_PREFIX`

Reply:
What does `SMART_SELECTS_URL_PREFIX` do exactly? From what I can tell, it prepend the autocomplete endpoints. Is that it?